src/components/vr_cooperation/src/base_command_request.cc

```diff
@@ -30,18 +30,38 @@
  POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include "vr_cooperation/commands/base_command_request.h"
 #include "utils/logger.h"
+#include "vr_cooperation/commands/base_command_request.h"
+#include "vr_cooperation/event_engine/event_dispatcher.h"
+#include "vr_cooperation/message_helper.h"
+#include "vr_cooperation/vr_module.h"
+#include "vr_cooperation/vr_module_constants.h"
 
 namespace vr_cooperation {
 
 namespace commands {
 
+using event_engine::EventDispatcher;
+
+using json_keys::kSuccess;
+using json_keys::kResultCode;
+using json_keys::kInfo;
+using json_keys::kId;
+using json_keys::kParams;
+using json_keys::kJsonrpc;
+using json_keys::kMethod;
+using json_keys::kResult;
+using json_keys::kCode;
+using json_keys::kError;
+using json_keys::kMessage;
+
 CREATE_LOGGERPTR_GLOBAL(logger_, "VRCooperation")
 
 BaseCommandRequest::BaseCommandRequest(
   const application_manager::MessagePtr& message)
   : message_(message) {
+  service_ = VRModule::instance()->service();
```

Comment:
after implement VRModule class functionality, we will remove singleton.
Giang changed it in https://github.com/CustomSDL/sdl_core/pull/175